ext/standard/uuencode.c

```diff
@@ -222,8 +222,8 @@ PHP_FUNCTION(convert_uudecode)
 	if (ZSTR_LEN(src) < 1) { RETURN_FALSE; }
 
 	if ((dest = php_uudecode(ZSTR_VAL(src), ZSTR_LEN(src))) == NULL) {
-		php_error_docref(NULL, E_WARNING, "The given parameter is not a valid uuencoded string");
-		RETURN_FALSE;
+		zend_value_error("The given parameter is not a valid uuencoded string");
+		return;
```

Comment:
I believe this change isn't okay. If you get a uuencoded string from an untrusted 3rd party, you'd currently want to write something like `if (false === $decoded = convert_uudecode($payload))` to check whether it was valid. This legitimate code would be broken by this change.

Reply:
Right, I'll remove this commit.